File: FormalBench-data/diverse/rule_14/SumNums.java
import java.io.*;
import java.lang.*;
import java.util.*;
import java.math.*;

class SumNums {

	public static int sumNums(int x, int y, int m, int n) {
		int sum = x + y;
		if (!(sum >= m && sum <= n)) {
			return sum;
		} else {
			return 20;
		}
	}
}

File: FormalBench-data/FilteredData/translated_c/seed_726_per_category_10_653ade686f/SumNums.c
#include <stdint.h>

int32_t sumNums(int32_t x, int32_t y, int32_t m, int32_t n) {
    int32_t sum = (int32_t)((uint32_t)x + (uint32_t)y);
    if (sum >= m && sum <= n) {
        return 20;
    } else {
        return sum;
    }
}
